<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WhatToTrackSelectDialog</class>
 <widget class="QDialog" name="WhatToTrackSelectDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>331</width>
    <height>190</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>SelectWhatToTrack</string>
  </property>
  <widget class="QFrame" name="frame">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>331</width>
     <height>191</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">
QFrame{
border: 3px solid gray;
border-radius: 40px;
background: white;
background: QLinearGradient(x1: 0, y1: 0, x2: 0, y2: 1, stop: 0 #eef, stop: 1 #ccf);
}

QRadioButton{

font-weight: bold;
font-size: 15px;

}

QFrame{
border: 3px solid gray;
border-radius: 40px;
background: white;
background: QLinearGradient(x1: 0, y1: 0, x2: 0, y2: 1, stop: 0 #eef, stop: 1 #ccf);
}

QLabel{
border: none;
background: white;
font-weight: bold;
font-size: 15px;

}
QLineEdit {
padding: 1px;
border-style: solid;
border: 2px solid gray;
border-radius: 8px;
}</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QRadioButton" name="radioButton_selectByMouse">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>110</y>
      <width>167</width>
      <height>19</height>
     </rect>
    </property>
    <property name="text">
     <string>Select By Myself</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButton_Pedestrian">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>20</y>
      <width>113</width>
      <height>19</height>
     </rect>
    </property>
    <property name="text">
     <string>Pedestrian</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButton_football">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>80</y>
      <width>95</width>
      <height>19</height>
     </rect>
    </property>
    <property name="text">
     <string>FootBall</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButton_car">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>50</y>
      <width>50</width>
      <height>19</height>
     </rect>
    </property>
    <property name="text">
     <string>Car</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_exit">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>150</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Close</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_apply">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>150</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Apply</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
